app/scripts/modules/core/src/loadBalancer/LoadBalancersTag.tsx

```diff
@@ -130,7 +130,7 @@ export class LoadBalancersTag extends React.Component<ILoadBalancersTagProps, IL
 
     return (
       <span className={className}>
-        {!isLoading && <Spinner size="nano" />}
+        {isLoading && <Spinner size="nano" />}
```

Comment:
Can probably omit this line altogether - the block on L112 should cover the spinner:
```javascript
    if (!totalCount) {
      return isLoading ? <Spinner size="nano" /> : null;
    }
```

Reply:
you are correct. we removed it +1